index.html

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
does this html file need to be checked in? or will the build automatically create this file? I hope I worked that questions so it makes sense!

Reply:
oh yeah! it's like Handlebars so it just builds it out. this one is called "Liquid"

Reply:
Okay 😄I think this file could be added to the `.gitignore` then? I will run the project though and double check that to make sure!